js/ssai_pulse.js

```diff
@@ -713,6 +757,43 @@ OO.Ads.manager(function(_, $)
       return _appendParamToUrl(url, SMART_PLAYER);
     }, this);
 
+    /**
+     * Parses the ad url to obtain the ssai guid and embed code
+     * @private
+     * @method SsaiPulse#_parseUrl
+     * @param {string} url The stream url
+     */
+    var _parseUrl = _.bind(function(url)
+    {
+      if (url === null) {
```

Comment:
If url is undefined we would still have an error thrown. It might be best to check if typeof url === "string" so that we can safely call .split in the next line.

Reply:
@aeng7 changed